### PR TITLE
New Port Luminous Slash

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Luminous Slash

## Game Information
- **Title**: Luminous Slash
- **URL**: https://gizmo199.itch.io/luminous-slash

## Submission Requirements
**CFW Tests:**
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

**Architecture**
[X] armHF
[X] Panfrost (Mainline)

**Resolution Tests**
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/luminousslash/Luminous Slash.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
```
Comment (Cebion) [inline ports/luminousslash/Luminous Slash.sh:3]: Old Port Header

File: ports/luminousslash/screenshot.png
Comment (Cebion) [inline ports/luminousslash/screenshot.png:1]: Screenshot should be 640x480.
You can take screenshots on your device for example with https://portmaster.games/port-tools.html#capture-video-and-screenshots

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/luminousslash/screenshot.png
Comment (Fraxinus88) [reply ports/luminousslash/screenshot.png:1]: Updated :)

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**